config/redirects

```diff
@@ -148,3 +148,20 @@ raw: docs/drivers/use-cases/storing-log-data -> ${devhub}/
 raw: docs/drivers/use-cases/pre-aggregated-reports-mmapv1 -> ${manual}/changeStreams/
 raw: docs/drivers/use-cases/pre-aggregated-reports -> ${manual}/changeStreams/
 raw: docs/drivers/tutorial/write-a-tumblelog-application-with-flask-mongoengine -> https://pymodm.readthedocs.io/en/latest/getting-started.html
+
+# Redirects for ecosystem that were previously handled in Fastly
+
+raw: docs/ecosystem/use-cases/pre-aggregated-reports -> ${devhub}/
+raw: docs/ecosystem/tutorial/authenticate-with-java-driver -> ${devhub}/
+raw: docs/ecosystem/use-cases/client-side-field-level-encryption-guide -> ${base}/security/client-side-field-level-encryption-guide/
+raw: docs/ecosystem/tools/http-interfaces -> ${devhub}/
+raw: docs/ecosystem/security/client-side-field-level-encryption-guide -> ${base}/security/client-side-field-level-encryption-guide/
+raw: docs/ecosystem/drivers?jump=docs -> ${base}/?jump=docs
```

Comment:
You can't have a query parameter in a redirect source

Reply:
haha fair